Feature use static id ary

tnagorra opened the pull request:
Fixes https://github.com/the-deep/deeper/issues/633
Fixes https://github.com/the-deep/deeper/issues/632
Fixes https://github.com/the-deep/deeper/issues/621
Fixes https://github.com/the-deep/deeper/issues/630
Fixes https://github.com/the-deep/deeper/issues/631
Fixes https://github.com/the-deep/deeper/issues/622
Fixes https://github.com/the-deep/deeper/issues/623

Fixes https://github.com/the-deep/deeper/issues/636
Fixes https://github.com/the-deep/deeper/issues/637
Fixes https://github.com/the-deep/deeper/issues/638
Fixes https://github.com/the-deep/deeper/issues/639
Fixes https://github.com/the-deep/deeper/issues/642
Fixse https://github.com/the-deep/deeper/issues/640

Depends on https://github.com/the-deep/server/pull/318
Depends on https://github.com/the-deep/server/pull/320

tnagorra committed:
Create constants to identify elements from assessment template

- Use new api for MultiViewContainer view

frozenhelium committed:
Add HNO and CNA tab in edit assessment page

frozenhelium committed:
Add UI for HNO and CNA

tnagorra committed:
Cleanup methodology identification after fixture

- Refactor schema to avoid mutation
- Add conditional case for HNO and CNA
- Fix proptypes for pending in each assessment tab

Add string for HNO/CNA title

tnagorra committed:
Add conditional schema for hno and cna

- Use Faram for hno and cna

AdityaKhatri commented on src/redux/selectors/siloDomainData/editAry.js:
```diff
@@ -497,6 +509,15 @@ export const assessmentSchemaSelector = createSelector(
             summary: createSummarySchema(focuses, selectedSectors, selectedFocuses),
             score: createScoreSchema(scorePillars, scoreMatrixPillars, selectedSectors),
         } };
+        console.warn(showHNO, showCNA);
```
Sir

AdityaKhatri commented on src/redux/selectors/siloDomainData/editAry.js:
```diff
@@ -497,6 +509,15 @@ export const assessmentSchemaSelector = createSelector(
+        if (showCNA) {
+            schema.fields.cna = createQuestionnaireSchema(aryTemplateQuestionnaire, 'cna');
+        }
+        console.warn(schema.fields.hno, schema.fields.cna);
```
Sir

tnagorra committed:
Add schema and computeSchema for hno and cna 'score'

- Add proptypes and LoadingAnimation for HNO and CNA
- Refactor computeSchema for Score tab
- Sort 'criteria' questions before 'ethos' questions
- Change question value from string to boolean
- Show calculate score for HNO and CNA

tnagorra committed:
Namespace cno and hna inside questionnaire

tnagorra committed:
Add hound config

AdityaKhatri commented on src/views/EditAry/RightPanel/Questionnaire/Question/index.js:
```diff
@@ -0,0 +1,56 @@
+    render() {
+        const {
+            className,
+            data,
```
Proptype not defined

AdityaKhatri commented on src/views/EditAry/RightPanel/Questionnaire/SubMethod/index.js:
```diff
@@ -0,0 +1,46 @@
+    render() {
+        const {
+            className,
+            data,
```
proptype not defined

AdityaKhatri commented on src/views/EditAry/RightPanel/Questionnaire/SubMethod/index.js:
```diff
@@ -0,0 +1,46 @@
+                </div>
+                <ListView
+                    className={styles.questionList}
+                    data={data.questions}
```
is data always defined?

AdityaKhatri commented on src/views/EditAry/RightPanel/Questionnaire/index.js:
```diff
@@ -0,0 +1,118 @@
+                                // FIXME: use strings
+                                title="Minimum Requirement"
+                                faramElementName="minimum-requirements"
+                                minValue={0}
```
use a variable for min and max value

tnagorra committed:
Treat use sector as a special case

- Show message according to use criteria, all quality criteria and
  minimum requirements
